jmix-reports/reports/src/main/java/io/jmix/reports/yarg/formatters/impl/XlsxFormatter.java

```diff
@@ -80,6 +99,8 @@ public class XlsxFormatter extends AbstractFormatter {
     protected Set<CellWithBand> innerFormulas = new HashSet<>();
     protected Set<CellWithBand> outerFormulas = new HashSet<>();
 
+    protected Map<String, List<XlsxImage>> templateImages = new HashMap<>();
```

Comment:
May be it would be better to move this field and all related logic to a separate class.

Reply:
moved static methods to util class